cosmo_tester/config_schemas/manager_image_names_centos.yaml

```diff
@@ -1,10 +1,22 @@
 namespace: manager_image_names_centos
 "master":
   description: Name of main manager image to test. This will be used in all manager tests.
-  default: cloudify-manager-premium-5.2.0
+  default: cloudify-manager-premium-5.3.0-.dev1
 "installer":
   description: Name of image with manager installer for bootstrap or cluster installs.
+  default: cfy-installer-premium-5.3.0-.dev1
+"5_2_0":
+  description: Name of manager image for 5.2.0. All non master images are only used for upgrade tests.
+  default: cloudify-manager-premium-5.2.0
+"5_2_0_installer":
+  description: Name of image with 5.2.0 manager installer for bootstrap or cluster installs.
   default: cfy-installer-premium-5.2.0
+"5_1_2":
+  description: Name of manager image for 5.1.2. All non master images are only used for upgrade tests.
+  default: cloudify-manager-premium-5.1.2
+"5_1_2_installer":
+  description: Name of manager image for 5.1.2. All non master images are only used for upgrade tests.
```

Comment:
Should be changed to " Name of image with 5.1.2 manager installer for bootstrap or cluster installs."